.github/workflows/fastapi.yml

```diff
@@ -0,0 +1,40 @@
+name: fastapi-testsuite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  fastapi-test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: fastapi
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: ddtrace
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - uses: actions/checkout@v2
+        with:
+          repository: tiangelo/fastapi
+          ref: 0.63.0
+          path: fastapi
+      - name: Install Flit
+        run: pip install flit
+      - name: Inject ddtrace
```

Comment:
We want to do something similar to what's done in the other workflows for checking out the repo and installing from the path since we want to run this workflow for a commit rather than what is currently deployed to pypi.